go/vt/binlog/binlog_streamer.go

```diff
@@ -791,7 +782,7 @@ func writeIdentifiesAsSQL(sql *bytes.Buffer, tce *tableCacheEntry, rs *mysql.Row
 		if valueIndex > 0 {
 			sql.WriteString(" AND ")
 		}
-		sql.WriteString(tce.ti.Columns[c].Name.String())
+		sql.Myprintf("%v", tce.ti.Columns[c].Name)
```

Comment:
No need to escape anything for column names, as we do for table names?

Reply:
`Myprintf("%v")` escapes column names. The code path will lead to `node.Format`. I hope I understood your question correctly.

Reply:
Yes, cool.